README.md

```diff
@@ -152,14 +147,66 @@ Use <kbd>Ctrl + Shift + F</kbd> to see auto-detected subtitle tracks for streami
 
 <kbd>Ctrl + Shift + F</kbd> also allows you to load an empty subtitle track so that you can extract audio and screenshots from streaming video without loading subtitles.
 
+### Filtering subtitle text
+
+If you'd like to filter out specific instances subtitle text, one way to do so is by using a regular expression (regex). asbplayer can match any sequence following a specified regex pattern and remove the matches.
+
+Under the MISC section in asbplayer settings, locate the "Subtitle regex filter" textbox. Enter an appropriate regex to filter desired content.
+You can replace filtered content similarly by entering a string into the "Subtitle regex filter text replacement" textbox. Leaving this blank will simply remove the content.
+
+Potential use cases for filtering include:
+ - Remove names enclosed by parenthesis to indicate speakers  (i.e. "**（山田）**　元気ですか？")
+ - Remove indications enclosed by brackets that sound or music that is playing (i.e. "**\[PLAYFUL MUSIC]**")
+
+Helpful materials on writing regular expressions:
+- [GeeksForGeeks regex tutorial](https://www.geeksforgeeks.org/write-regular-expressions/)
+- [RexEgg regex cheatsheet](https://www.rexegg.com/regex-quickstart.html)
+- Test regex with the [Regexr webtool](https://regexr.com/)
+
+> Note: Japanese parentheses are usually different characters than the ones found on english keyboards: "()" versus "（）"

```

Comment:
I think this note about parantheses can be removed. Many people reading this don't care about Japanese and I'd prefer asbplayer to be seen as language-agnostic.